certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -2291,7 +2291,14 @@
             "Delivered",
             "Completed",
             "Declined",
-            "Voided"
+            "Voided",
+			"envelope-resent",
+			"envelope-corrected",
+			"envelope-purge",
+			"envelope-deleted",
+			"envelope-discard",
+			"click-agreed",
```

Comment:
is this the entire list given by Marshall ? 
or we are doing only a subset of what he asked for.

Reply:
Yes, this list covers all the envelope and click events that DocuSign REST API supports.